bubble_sort.py

```diff
@@ -1,36 +1,18 @@
-"""
-Módulo que guarda as funções para o Bubble Sort
-"""
 
+def bubble_sort(v):
+    fim = len(v)
+    while fim > 0:
+        i = 0
+        #percorrendo um vetor de 0 até fim
+        while i < fim-1:
+            if v[i] > v[i+1]:
+                #realizando a troca da posição atual pela próxima
+                temp = v[i]
+                v[i] = v[i+1]
+                v[i+1] = temp
+            i += 1
+        fim -= 1
+
+    return 200.000
```

Comment:
Não entendi porque vocês estão retornando o valor `200` (lembrem-se que o `.` é separador de vírgula no Python)
Aqui vocês teriam que retornar a própria lista depois de ordenada.

```suggestion
    return v
```